### PR TITLE
Diffusers, txt2img and img2img, make sure guidance scale defaults to 0 when num steps <=4

### PR DESCRIPTION
**Pull request by oOraph**

### CODE REVIEW

File: docker_images/diffusers/app/pipelines/text_to_image.py [hunk 1]
```diff
@@ -169,6 +169,9 @@ def _process_req(self, inputs, **kwargs):
                 kwargs["num_inference_steps"] = 20
             # Else, don't specify anything, leave the default behaviour
 
+        if int(kwargs.get("num_inference_steps", 20)) <= 4 and 'guidance_scale' not in kwargs.keys():
+            kwargs["guidance_scale"] = 0
+
```
Comment (apolinario) [inline docker_images/diffusers/app/pipelines/text_to_image.py:174]:
```suggestion
        if "guidance_scale" not in kwargs:
            default_guidance_scale = os.getenv("DEFAULT_GUIDANCE_SCALE")
            if default_num_steps:
                kwargs["guidance_scale"] = int(default_guidance_scale)
            # Else, don't specify anything, leave the default behaviour
```

File: docker_images/diffusers/app/lora.py [hunk 2]
```diff
@@ -101,9 +101,11 @@ def _load_textual_embeddings(self, adapter, model_data):
...
             embedding_path = self._hub_repo_file(
                 repo_id=adapter,
-                filename="embeddings.safetensors"
-                if self._is_safetensors_pivotal(model_data)
-                else "embeddings.pti",
+                filename=(
+                    "embeddings.safetensors"
+                    if self._is_safetensors_pivotal(model_data)
+                    else "embeddings.pti"
+                ),
```
Comment (apolinario) [inline docker_images/diffusers/app/lora.py:108]: Seems unrelated?

Comment (oOraph) [reply docker_images/diffusers/app/lora.py:108]: yes -> `black .` reformatting made this on my laptop, removing, probably a lib version difference
